### PR TITLE
Bring divergent release back into the fold

### LINKED ISSUE #6: Do something about divergent tag that's not part of master
Tag `2.1.0` is part of a sequence of commits that diverged from the `master` branch back in 2017.

This makes it ambiguous to determine how to label our own recent changes to `master`.

Either merge these divergent changes or throw them away and delete the tag.

### PR DESCRIPTION
**Pull request by g-andrade**
Closes #6.

### CODE REVIEW

File: eqc/pbkdf2_eqc.erl [hunk 1]
```diff
@@ -0,0 +1,82 @@
+-module(pbkdf2_eqc).
```
Comment (paulo-ferraz-oliveira) [inline eqc/pbkdf2_eqc.erl:1]: I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

File: rebar.config [hunk 2]
```diff
@@ -3,22 +3,15 @@
...
-%% == xref ==
+{eunit_opts, [verbose]}.
 
-%% Enable xref warnings.
-{xref_warnings, true}.
+{plugins, [{eqc_rebar, {git, "https://github.com/Quviq/eqc-rebar", {branch, "master"}}}]}.
 
-%% xref checks to run
-{xref_checks, [undefined_function_calls]}.
+{xref_checks,[undefined_function_calls,undefined_functions,locals_not_used,
```
Comment (paulo-ferraz-oliveira) [inline rebar.config:16]: We can do without `undefined_functions` and `deprecated_functions`.

File: test/josefsson-draft-vectors.config [hunk 3]
```diff
@@ -0,0 +1,34 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```
Comment (paulo-ferraz-oliveira) [inline test/josefsson-draft-vectors.config:1]: I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

File: test/rfc3962-vectors.config [hunk 4]
```diff
@@ -0,0 +1,50 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```
Comment (paulo-ferraz-oliveira) [inline test/rfc3962-vectors.config:1]: I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

File: test/rfc6070-vectors.config [hunk 5]
```diff
@@ -0,0 +1,28 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```
Comment (paulo-ferraz-oliveira) [inline test/rfc6070-vectors.config:1]: I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

File: test/unofficial-vectors.config [hunk 6]
```diff
@@ -0,0 +1,37 @@
+%% -*- mode: erlang; erlang-indent-level: 4; indent-tabs-mode: nil -*-
```
Comment (paulo-ferraz-oliveira) [inline test/unofficial-vectors.config:1]: I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

File: test/vector_tests.erl [hunk 7]
```diff
@@ -0,0 +1,96 @@
+%% -------------------------------------------------------------------
```
Comment (paulo-ferraz-oliveira) [inline test/vector_tests.erl:1]: I'm assuming this file is left untouched except for the occasional static analysis -related fix. Is my assumption correct?

**Comment by paulo-ferraz-oliveira** [comment]
I'm OK with the general direction of this pull request. Only have a couple of questions and one request for change.

File: rebar.config [hunk 8]
```diff
@@ -3,22 +3,15 @@
...
+    deprecated_function_calls, deprecated_functions]}.
```
Comment (paulo-ferraz-oliveira) [inline rebar.config:17]:
```suggestion
    deprecated_function_calls]}.
```

Comment (paulo-ferraz-oliveira) [inline rebar.config]: We can do without `undefined_functions` and `deprecated_functions`.
```suggestion
{xref_checks,[undefined_function_calls,locals_not_used,
    deprecated_function_calls]}.
```

**Review by paulo-ferraz-oliveira — APPROVED** [review]

File: eqc/pbkdf2_eqc.erl (hunk 1 above)
Comment (g-andrade) [reply eqc/pbkdf2_eqc.erl:1]: In the sense that we won't run the tests contained within it? Sure.

File: test/josefsson-draft-vectors.config (hunk 3 above)
Comment (g-andrade) [reply test/josefsson-draft-vectors.config:1]: I don't understand your question - this is test data.

File: test/rfc3962-vectors.config (hunk 4 above)
Comment (g-andrade) [reply test/rfc3962-vectors.config:1]: I don't know... is it? This is test data.

File: test/rfc6070-vectors.config (hunk 5 above)
Comment (g-andrade) [reply test/rfc6070-vectors.config:1]: I'm not sure. Do static analysis fixes ever apply to test data?

File: test/unofficial-vectors.config (hunk 6 above)
Comment (g-andrade) [reply test/unofficial-vectors.config:1]: I'm assuming static analysis doesn't apply here, given that this is test data.

File: test/vector_tests.erl (hunk 7 above)
Comment (g-andrade) [reply test/vector_tests.erl:1]: Presumably, yes.

**Comment by g-andrade** [comment]
Ready to merge if you're satisfied with my answers to your questions.

**Pull request merged**